### ISSUE TITLE
Graph view elements unreadable in dark theme

### ISSUE THREAD

**Issue by LarsKemmann** [microsoft/AzureStorageExplorer#292]
**Storage Explorer Version**: 1.2.0
**Platform/OS Version**: Windows 10 x64 (1803)
**Architecture**: ia32

#### Steps to Reproduce: ####
1. Choose dark theme
2. Run a query against a Cosmos DB graph database

#### Expected Experience: ####
- Vertex labels should show in a light font color so they can be read against the black background
- Edit pencil icons on Properties & Sources should show up in a light color
- Progress indicator circles should be light so they can be seen

#### Actual Experience: ####
- Vertex labels are shown in a dark font color and cannot be read against the black background
- Edit pencil icons can't be seen
- Progress indicators are black so only the gray edges of the circles are visible

**Comment by ziyel** [comment]
Thanks for the feedback @LarsKemmann. We will fix this.

**Comment by aliuy** [comment]
Azure Storage Explorer support for Azure Cosmos DB has been discontinued as of v1.24.0. We recommend users to use Azure Cosmos DB’s Data Explorer, which can be found in [Azure Portal](https://portal.azure.com/) or as a stand-alone version in http://cosmos.azure.com/ for the best Azure Cosmos DB experience. You can also find additional Azure Cosmos DB tooling here: [Azure Cosmos DB Tools](https://developer.azurecosmosdb.com/tools)
